.github/workflows/ci.yaml

```diff
@@ -79,7 +79,7 @@ jobs:
           echo "Github action is ${{ github.event.action }}." >> $GITHUB_STEP_SUMMARY
 
   build:
-    runs-on: selfhosted-ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
I would suggest pinned version of Ubuntu would be better as build process does get inconsistent with latest